src/components/App/App.js

```diff
@@ -7,35 +7,34 @@ import RecyclingTips from '../RecyclingTips/RecyclingTips';
 import SearchBar from '../SearchBar/SearchBar';
 import Footer from '../Footer/Footer';
 import './App.css';
-import Creators from '../Creators/Creators'
-
-// const Creators = React.lazy(() => import("./pages/Creators"));
+import Creators from '../Creators/Creators';
+import NoMatch from '../NoMatch/NoMatch';
 
 const App = () => {
-  const [placeIds, setPlaceIds] = useState([]);
+  const [locationDetails, setlocationDetails] = useState([]);
 
-  const updatePlaceIds = (newPlaceIds) => {
-    setPlaceIds(newPlaceIds);
-    console.log('updatePlaceIds');
+  const updateLocationDetails = (newLocationDetails) => {
+    setlocationDetails(newLocationDetails);
   };
 
   return (
     <main className="app">
       <NavBar />
       <Routes>
         <Route 
-          path="/"
+          path="recycle-me-ui"
```

Comment:
Why were we unable to use just a / to route to our application?

Reply:
I think this has something to do with deployment setting the base url to end with `/recycle-me-ui`. I'll add this to the backlog.